File: Backend-Dataaccess/src/Objects/Room.java
package Objects;

public class Room {
    private String address;
    private int price;

    public String getAddress() {
        return address;
    }
    public int getPrice() {
        return price;
    }
    public void setAddress(String address) {
        this.address = address;
    }
    public void setPrice(int price) {
        this.price = price;
    }
    public Room(String address, int price) {
        this.address = address;
        this.price = price;
    }
}

File: Website/BBeu-Website/src/app/Objects/room.ts
export class Room {


  // public address!: string;
  // public price!: number;
  //
  // constructor(address: string, price: number){
  //   // console.log('construktortesting');
  //   this.address = address;
  //   // console.log(address);
  //
  //   this.price = price;
  // }

  private _address!: string;
  private _price!: number;
  private _testee: number = -3;

  constructor(address: string, price: number){
    // console.log('construktortesting');
    this._address = address;
    // console.log(address);

    this._price = price;
  }

  public getaddress(){
    return this._address;

  }
  public getprice(): number {
    return this._price;
  }
  // public setaddress(value: string) {
  //   this._address = value;
  // }
  // public setprice(value: number) {
  //   this._price = value;
  // }


  getRoom():string{
    return '&{this.Address}&{this.Price}'
  }
  toJSON() {
    return {getaddress: this._address,getprice: this._price};
  }

  public testing(){

  }
}
